### ISSUE TITLE
[CI] :plugins:discovery-gce:qa:gce:integTestCluster#wait failing in 6.x

### ISSUE THREAD

**Issue by droberts195** [elastic/elasticsearch#34272]
On the 6.x branch the cluster started by the :plugins:discovery-gce:qa:gce:integTest task is failing to start up.

The error is:

```
14:16:55 |    [2018-10-03T14:16:54,667][WARN ][o.e.c.g.GceInstancesServiceImpl] [node-2] Problem fetching instance list for zone test-zone
14:16:55 :plugins:discovery-gce:qa:gce:integTestCluster#node2.stop (Thread[Task worker for ':' Thread 12,5,main]) completed. Took 0.003 secs.
14:16:55 |    com.google.api.client.googleapis.json.GoogleJsonResponseException: 500 Internal Server Error
14:16:55 |    /var/lib/jenkins/workspace/elastic+elasticsearch+6.x+intake/plugins/discovery-gce/qa/gce/build/generated-resources/nodes.uri
```

Some failed builds are:

* https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+intake/117/console
* https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+intake/118/console
* https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+intake/119/console

I will mute this task.

**Comment by droberts195** [comment]
This test task is disabled in 6.x by 9055797088113393f8c9759fa5abb16ca88e768d

**Comment by vladimirdolzhenko** [comment]
Gonna run CI for `fix/34272` before merging that to 6.x

**Comment by vladimirdolzhenko** [comment]
fixed in 3d5a386 : https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+intake/124/
